src/content/learn/importing-and-exporting-components.md

```diff
@@ -37,7 +37,7 @@ function Profile() {
 export default function Gallery() {
   return (
     <section>
-      <h1>Amazing scientists</h1>
+      <h1>Ilmwuan Hebat</h1>
```

Comment:
Masih typo 😭
```suggestion
      <h1>Ilmuwan Hebat</h1>
```

Reply:
Shouldn't it be `Ilmuwan-ilmuwan` or `Para ilmuwan` because the source is `scientists` not just `scientist`?

```suggestion
      <h1>Ilmuwan-ilmuwan` Hebat</h1>
```

```suggestion
      <h1>Para ilmuwan Hebat</h1>
```

Reply:
I'm sorry, gonna fix that